### PR TITLE
Update the ScopeSpecification proto

### PR DESCRIPTION
**Pull request by dwedul-figure**
## Description

Change the ScopeSpecification proto to better represent what's needed.

closes: #71

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/provenance-io/provenance/blob/main/CONTRIBUTING.md#pr-targeting))
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] ~~Wrote unit and integration [tests](https://github.com/provenance-io/provenance/blob/main/CONTRIBUTING.md#testing)~~ - n/a: only updating data models.
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [x] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [x] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: proto/provenance/metadata/v1/specification.proto [hunk 1]
```diff
@@ -91,6 +100,22 @@ message ContractSpec {
...
+message Info {
+  // A Name for this thing.
+  string name = 1;
+  // A description of this thing.
+  string description = 3;
+  // URL to find even more info.
+  string website_url = 4 [
+    (gogoproto.moretags)   = "yaml:\"website_url\""
+  ];
+  // URL of an icon for this. thing.
```
Comment (iramiller) [inline proto/provenance/metadata/v1/specification.proto:113]: nit: Extra period.  Could be a bit more clear.

**Review by iramiller — COMMENTED** [review]
Very nice.

File: proto/provenance/metadata/v1/specification.proto (hunk 1 above)
Comment (dwedul-figure) [reply proto/provenance/metadata/v1/specification.proto]: Fixed -> "// URL of an icon."

**Review by iramiller — APPROVED** [review]
LGTM.

**Pull request merged**